src/main/java/com/fivetran/agent/mysql/output/BucketOutput.java

```diff
@@ -146,7 +123,7 @@ private void flushToBucket() throws IOException {
     }
 
     private void refreshFileChannel() {
-        path = Paths.get(DATA_FILE_PREFIX + Instant.now().getEpochSecond() + DATA_FILE_EXTENSION);
+        path = Paths.get(DATA_FILE_PREFIX + Instant.now().toEpochMilli() + DATA_FILE_EXTENSION);
```

Comment:
We shouldn't have to sleep all those test threads in MysqlUpdaterV4/MysqlAgentUpdater specs